add logs and metrics preview form (#30400)
DataDog/documentation: content/en/ddsql_editor/_index.md

GROUP BY instance_type
{{< /code-block >}}

## Explore your telemetry

> View and filter the list of tables and fields in the schema side panel:

Reviewer: ```suggestion
View, filter, and built queries in the Data Explorer.
```

{{< img src="ddsql_editor/data-tab.png" alt="The data tab showing table information for aws.ec2_instance" style="width:70%;" >}}

> For Logs and Metrics, use the query builder to help you generate table functions:
> 
> {<img width="414" height="116" alt="image" src="https://github.com/user-attachments/assets/09dc368e-5483-413f-bfd5-5fc9bb8e89e2" />}

Reviewer: I added a line to the first sentence about this functionality. We can expand it to a separate section if you think users need more guidance on how to build queries after GA.
```suggestion
```

Author: see my comment above. No need for seperate section but I wanted to call this out seperately, as a unique workflow to Logs and Metrics

but still leave the above as general info about the data explorer

Reviewer: ```suggestion
For Logs and Metrics, use the query builder to help you generate table functions.
```
